src/main/resources/assets/computercraft/lua/rom/programs/edit.lua

```diff
@@ -31,12 +31,12 @@ local bRunning = true
 -- Colours
 local highlightColour, keywordColour, commentColour, textColour, bgColour, stringColour
 if term.isColour() then
```

Comment:
Not quite: you've removed the check in the bios, but here and in the other programs you only load the colours from settings on an advanced computer - normal ones still use the default.